### PR TITLE
updated read-me with instructions on how to run app

### PR DESCRIPTION
**Pull request by quichey**
also removed extraneous edit.html.haml file in root directory

### CODE REVIEW

**Comment by mihizi** [comment]
Why did you delete edit.html.haml? 😨

File: README.rdoc [hunk 1]
```diff
@@ -4,8 +4,64 @@ Link to our Web App: https://entrepreneurs-at-berkeley.herokuapp.com/
...
+
+Then create the file config/application.yml and add the following lines (to create environment variables):
+
+  SERVER_ADDRESS: smtp.gmail.com
+  EMAIL_DOMAIN: gmail.com
+  EMAIL_USERNAME: (1)
+  EMAIL_PASSWORD: (2)
+  REGULAR_CODE: (3)
+  ADMIN_CODE: (4)
+
```
Comment (georgelzeng) [inline README.rdoc]: Also needs block for SUPERADMIN_CODE

Comment (georgelzeng-zz) [inline README.rdoc:58]: Also needs space for SUPERADMIN_CODE

**Review by georgelzeng-zz — CHANGES_REQUESTED** [review]
One last item

**Comment by quichey** [comment]
to mihir: I deleted an edit.html.haml that was created in the root directory. It was mistakenly created by me while i was in atom. The original edit.html.haml still exists.

File: README.rdoc (hunk 1 above)
Comment (quichey) [reply README.rdoc:58]: good catch. added

**Review by georgelzeng-zz — APPROVED** [review]
all g

**Pull request merged**